prestodb/hdp2.6-hive-kerberized/Dockerfile

```diff
@@ -23,6 +23,9 @@ COPY ./files /
 # CREATE KERBEROS DATABASE
 RUN /usr/sbin/kdb5_util create -s -P password
 
+# CREATE ANOTHER KERBEROS DATABASE
+RUN /usr/sbin/kdb5_util create -d /var/kerberos/krb5kdc/principal-other -r OTHERLABS.TERADATA.COM -s -P password
```

Comment:
nit: `123456` is a better password than `password` (at usage site, the meaning is easier to guess)